packages/react-router/src/runtime.ts

```diff
@@ -58,29 +62,32 @@ export class Runtime extends AbstractRuntime<RootRoute | Route, RootNavigationIt
+            if (pendingRegistrations.has(ManagedRoutes.$name!)) {
+                // eslint-disable-next-line max-len
+                throw new Error("[squide] The \"ManagedRoutes\" outlet route is missing from the router configuration. The \"ManagedRoutes\" outlet route must be added as a children of an hoisted route. Did you forget to include the \"ManagedRoutes\" outlet route or hoist the parent route that includes the \"ManagedRoutes\" outlet route?");
+            }
 
+            let message = `[squide] ${pendingRegistrations.size} parent route${pendingRegistrations.size > 0 ? "s" : ""} were expected to be registered but ${pendingRegistrations.size > 0 ? "are" : "is"} missing:\r\n\r\n`;
             let index = 0;
 
             // It's easier to use for ... of with a Map object.
-            for (const [layoutPath, nestedRoutes] of pendingRegistrations) {
+            for (const [parentId, nestedRoutes] of pendingRegistrations) {
                 index++;
 
-                message += `${index}/${pendingRegistrations.size} Missing layout path: "${layoutPath}"\r\n`;
+                message += `${index}/${pendingRegistrations.size} Missing parent route with the following path or name: "${parentId}"\r\n`;
                 message += "    Pending registrations:\r\n";
 
                 for (const x of nestedRoutes) {
-                    message += `        - "${x.index ? `${layoutPath} (index)` : x.path}"\r\n`;
+                    message += `        - "${x.path ?? x.$name ?? "(no identifier)"}"\r\n`;
                 }
 
                 message += "\r\n";
             }
 
-            message += `If you are certain that the layout route${pendingRegistrations.size !== 1 ? "s" : ""} has been registered, make sure that the following conditions are met:\r\n`;
-            message += "- The missing nested layout routes path property perfectly match the provided \"layoutPath\" (make sure that there's no leading or trailing \"/\" that differs).\r\n";
-            message += "- The missing nested layout routes has been registered with the \"registerRoutes()\" function. A route cannot be registered under a nested layout route that has not be registered with the \"registerRoutes()\" function.\r\n";
-            message += "- If a nested layout route is an index route, make sure that \"/$index$\" string has been appended to the \"layoutPath\".\r\n\r\n";
-            message += "For more information about nested layout routes, refers to https://gsoft-inc.github.io/wl-squide/reference/runtime/runtime-class/#register-routes-under-a-specific-nested-layout-route.\r\n";
+            message += `If you are certain that the parent route${pendingRegistrations.size > 0 ? "s" : ""} has been registered, make sure that the following conditions are met:\r\n`;
```

Comment:
pendingRegistrations.size === 1? "" : "s"

0 has an s

Reply:
line 70 has the same check

Reply:
Oopsy!